src/main/java/com/vladyslavpalamarchuk/trackmycoin/domain/Command.java

```diff
@@ -0,0 +1,21 @@
+package com.vladyslavpalamarchuk.trackmycoin.domain;
+
+import lombok.Getter;
+
+
+@Getter
+public enum Command {
+
+    START("/start", "Welcome to TrackMyCoin 📈💰!\nThis bot helps you track cryptocurrency prices\nand will send you notifications when your target price \nis reached.\n\nFor more information about the bot, type /info."),
```

Comment:
Please move all static values (text) to application.yml

Reply:
Also, please create task in Jira to move this values from application.yml to database. It will increase flexibility in changes for us